demo/3001-shop/pages/_app.js

```diff
@@ -1,16 +1,78 @@
-require('@module-federation/nextjs-mf/lib/include-defaults');
```

Comment:
why remove include defaults? these need to be imported somewhere otherwise webpack will tree shake unused modules that must come from host

Reply:
Tnx. I made a mistake in the build diagram.  `nextjs-mf/lib/include-defaults` is added by `patchNextClientPageLoader` webpack loader in the main compiler. Implicit import of `include-defaults` helps end-users NOT change their _app.tsx with additional import – few imports few questions. And it does not tree shook by webpack, because webpack resolves all imports in the text that was added via `patchNextClientPageLoader`.

<img width="901" alt="Screen Shot 2022-09-15 at 10 52 05" src="https://user-images.githubusercontent.com/1946920/190317406-c306688f-3a2b-4ec6-9ff5-4855df17f685.png">

Reply:
Yep, it imported in `patchNextClientPageLoader`. 
So developers may not do it explicitly anymore. 
But they will be forced to import `include-defaults` only if set `extraOptions.disableMFClient: true` in next.config.js.

<img width="1091" alt="Screen Shot 2022-09-15 at 11 22 18" src="https://user-images.githubusercontent.com/1946920/190321156-b8d8972f-5ec7-4ecb-9683-8b1772c00a21.png">

Reply:
even better, well done